src/modules/mavlink/mavlink_main.h

```diff
@@ -177,6 +177,12 @@ class Mavlink
 		BROADCAST_MODE_ON
 	};
 
+	enum FLOW_CONTROL_MODE {
```

Comment:
What if you used a class enum here? Then you'd use FLOW_CONTROL_MODE::ON instead of generic ON/OFF in the namespace.

Reply:
fixed